SystemInfo/Collector/DoctrineDatabaseSystemInfoCollector.php

```diff
@@ -39,11 +46,24 @@ public function __construct(Connection $db)
      */
     public function collect()
     {
+        $this->connection = $this->connectionList[$this->siteAccess->name];
```

Comment:
IMO it doesn't seem necessary to use a `connection` property if it's not used elsewhere, could just as well use a simple variable.